bundles/org.eclipse.jface/src/org/eclipse/jface/wizard/WizardDialog.java

```diff
@@ -1572,4 +1574,10 @@ public void run() {
 			});
 		}
 	}
+
+	@Override
+	protected IDialogSettings getDialogBoundsSettings() {
+		IDialogSettings dialogSettings = getWizard().getDialogSettings();
+		return DialogSettings.getOrCreateSection(dialogSettings, getClass().getSimpleName());
```

Comment:
You need to check for null here, also choosing a section based on the classname might clash and one better choose something that appends a key (e.g. 
```suggestion
		return DialogSettings.getOrCreateSection(dialogSettings, getClass().getSimpleName()+".dialogBounds");
```

Reply:
done